x-pack/plugins/fleet/common/services/agent_policies_helpers.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
```

Comment:
@elastic/kibana-core  we had to add the space plugin as required for this, do you think it make sense or we should not use the constant here ?